notebooks/utide_real_data_example.ipynb

```diff
@@ -105,18 +125,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The utide package works with ordinary numpy arrays, not with Pandas Series or Dataframes, so we need to make a `time` variable in floating point days since a given epoch, and use the `values` attribute of the elevation anomaly (a Pandas Series) to extract the underlying numpy ndarray."
+    "The utide package works with ordinary numpy arrays, not with Pandas Series or Dataframes, so we need to make a `time` variable in floating point days since a given epoch, and use the `values` attribute of the elevation anomaly (a Pandas Series) to extract the underlying numpy ndarray.\n",
```

Comment:
The cell is updated as following - 

From **utide v2.7** pandas `datetime` index can be passed directly as the time variable. For elevation or current, utide only works with ordinary numpy arrays, so we use the `to_numpy()` method of the elevation anomaly (a Pandas Series) to extract the underlying numpy ndarray.

Note: The utide package before 2.7 worked only with a `time` array in floating point days since `0001-01-01`.